jekyll/_includes/getting-started-section-header.html

```diff
@@ -0,0 +1,9 @@
+<div class='section-header-wrapper'>
+  <h2 class='section-title'>{{include.title}}</h2>
+  <img class='image-banner' src="{{include.imagePath}}" alt="header banner" />
+  <div class='content-wrapper'>
+    <p>
+      {{include.content}}
+    </p>
+  </div>
```

Comment:
if content could be anything, I don't think it should be in a `<p>`?

Reply:
good call, I can update it to a `<span>`?